components/Dropdown/Dropdown.js

```diff
@@ -2,27 +2,29 @@ class Dropdown {
   constructor(element) {
     
     // Assign this.element to the dropdown element
-    this.element;
+    this.element = element;
     
     // Get the element with the ".dropdown-button" class found in the dropdown element (look at the HTML for context)
-    this.button = this.element.querySelector();
+    this.button = this.element.querySelector('.dropdown-button');
     
     // assign the reference to the ".dropdown-content" class found in the dropdown element
-    this.content;
+    this.content = document.querySelector('.dropdown-content');
```

Comment:
I like how you used document here and this.element. Document works here because there is only one `.dropdown-content`. 

If there were multiple, you would want to query select the specific `.dropdown-content` or select them all and iterate through them + use a data attribute or something.